the-way-we-work.md

```diff
@@ -0,0 +1,664 @@
+
+
+### 3.3 Growing the Team
+
+**Code Reviews**
+
+This work will be carried out within a delivery team.
+
+You will:
+
+- agree a code review checklist with your team
+- invest enough time to provide a thorough review
+- review other people's code in before writing your own
+- use the code review as an opportunity to provide constructive advice and teaching
+
+**Mentoring and Pairing**
+
+This work will be carried out within a delivery team.
+
+You will:
+
+- actively seek out opportunities to develop the skills of other developers
+- pair with other developers whenever possible to complete feature tickets
+
+**Coaching**
+
+This work will be carried out on a one-to-one basis by arrangement with other developers.
+
+You will:
+
+- design programmes of coaching to improve developers' knowledge of agile software development
+- bprovide one-to-one coaching to support developers through their immediate and specific work issues
+- promote a coaching culture in NHSUK software development
+
+**Delegation**
+
+This work will be carried out in collaboration with the leadership team.
+
+You will:
+
+- identify opportunities to stretch and empower other members of the leadership team
+- delegate responsibility for the completion of significant work packages (e.g. lean improvement projects)
+- ensure resources are made available as agreed (e.g. developer time)
+- provide support and guidance where requested or required
+- champion the activities of other members of the leadership team (e.g. via all-hands presentations)
+
+
+### 3.4 Duty of Care
+
+**Buddy to Apprentice**
+
+This work will be carried out TODO
+
+You will:
+
+- bullets
+- bullets
+- bullets
+
+**Line Management and Recruitment of Band 6/7**
+
+This work will be carried out on an ongoing basis.
+
+You will:
+
+- comply with corporate policies and procedures
+- meet with each of your reports once each week to discuss and record progress towards objectives and key results
+- ensure each of your reports is carrying out their role as described in this document
+- participate in recruitment activities and interview panels
+
+**Line Management and Recruitment of Band 8a and Contractors**
+
+This work will be carried out on an ongoing basis.
+
+You will:
+
+- comply with corporate policies and procedures
+- meet with each of your reports once each week to discuss and record progress towards objectives and key results
+- ensure each of your reports is carrying out their role as described in this document
+- participate in recruitment activities and interview panels
+- ensure your reports are appropriately managing their own reports
+
+**Line Management and Recruitment of Band 8b and Contractors**
+
+This work will be carried out on an ongoing basis.
+
+You will:
+
+- comply with corporate policies and procedures
+- meet with each of your reports once each week to discuss and record progress towards objectives and key results
```

Comment:
I was thinking once a week is an over kill and it might be worth meeting once a sprint. Allows the report to give more of a report back of what they have achieved in that sprint rather than each week

Reply:
I will change this to "on the first day of each sprint". This addresses your comment above, and it seems to me that the first day of a new sprint is the best time to look back at the previous week while also discussing which OKRs they will look at in the new sprint.